view/base/web/error-handler.js

```diff
@@ -26,6 +26,43 @@
         sendTimer = window.setTimeout(sendError, sendDelay);
     };
 
+
+    
+    // Override console.error method so these errors are also logged
+
+    var originalConsole = {
+        error: console.error
+    };
+
+
+    function logToQueue(type, args) {
+        if (sendQueue.length > 100) {
+            // That's a lot of errors! Let's not overwhelm the server with more.
+            return;
+        }
+        if (!sendQueue.length) {
+            sendTimer = window.setTimeout(sendError, sendDelay);
+        }
+        sendQueue.push({
+            browser: {
+                height: window.innerHeight,
+                url: window.location.href,
+                width: window.innerWidth,
+            },
```

Comment:
This block (and `event.timer`) below could be part of the shared-functionality function.